plugin/trino-hudi/src/main/java/io/trino/plugin/hudi/HudiUtil.java

```diff
@@ -161,4 +175,47 @@ public static FileStatus getFileStatus(HoodieBaseFile baseFile)
             throw new TrinoException(HUDI_CANNOT_OPEN_SPLIT, "Error getting file status of " + baseFile.getPath(), e);
         }
     }
+
+    public static String convertToHoodieInstantTimeFormat(String userInputFormat, String timeStamp)
+    {
+        try {
+            if (!DEFAULT_INSTANT_TIME_FORMAT.equals(userInputFormat)) {
+                SimpleDateFormat formatter = new SimpleDateFormat(userInputFormat);
+
+                Date userInputTime = formatter.parse(timeStamp);
+                formatter.applyPattern(DEFAULT_INSTANT_TIME_FORMAT);
+                return formatter.format(userInputTime);
+            }
+            else {
+                return timeStamp;
+            }
+        }
+        catch (ParseException e) {
```

Comment:
`ParseException` won't be thrown then the condition goes to `else`. Please minimize the try-catch block. 
Also,